src/components/__tests__/TopNav.test.tsx

```diff
@@ -1,65 +1,62 @@
+/**
+ * @jest-environment jsdom
```

Comment:
Hey I notice you declare "jsdom" on many of these tests files, if you find yourself using a particular jest preset often you can alternatively set the preset globally, this can be done by adding the following code to jest.config.ts:
module.exports = {
    preset: "jsdom",
};

Reply:
Thanks for that, good to know!